keyboards/nyquist/keymaps/danielhklein/keymap.c

```diff
@@ -7,32 +7,41 @@
  Major changes made:
   - DVORAK, COLEMAK, FUNCTION, MOUSE, and ADJUST layers have been removed
   - right 2u key performs backspace, not enter
-  - no right ctrl key
-  - direction keys added to bottom row
   - LOWER layer controls function keys, media controls, and underglow
+  - ARROW layer uses JKLI for arrow keys
+  - Bootmagic enabled and bootmagic initialization key changed from space to enter
 */
+
 extern keymap_config_t keymap_config;
 
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
 enum custom_layers {
   _QWERTY,
   _LOWER,
-  _RAISE
+  _RAISE,
+  _ARROW
 };
 
 enum custom_keycodes {
   QWERTY = SAFE_RANGE,
   LOWER,
-  RAISE
+  RAISE,
+  ARROW
 };
 
   // Enable these functions using FUNC(n) macro.
   const uint16_t PROGMEM fn_actions[] = { //ACTION_LAYER_TAP_TOGGLE requires that number of taps be defined in *config.h* - default set to 5
-      [0] = ACTION_LAYER_TAP_KEY(_LOWER, KC_SPC),    //Hold for momentary Lower layer, Tap for Space,
-      [1] = ACTION_LAYER_TAP_KEY(_RAISE, KC_BSPC)     //Hold for momentary Raise layer, Tap for Backspace,
+      [0] = ACTION_LAYER_TAP_KEY(_LOWER, KC_SPC),    //Hold for momentary Lower layer, Tap for Backspace, 
```

Comment:
You could us `LT(_LOWER, KC_SPC)` in the keymap instead, if you wanted. 

(this is optional, as it does the same thing)

Reply:
Fixed.